### PR TITLE
new tips, updated old tips

### PR DESCRIPTION
**Pull request by plapatin**
## About The Pull Request
Exactly what it says on the tin, again. With the recent influx of new players I figured it'd be a good idea to sift through the tips.txt and update outdated tips and in some new ones.
Also removes clockcult tips, honk honk

## Why It's Good For The Game
Information is ultimately good for all of us.

## Changelog
:cl:
add: Watch the Tip of the Round messages, there's a few extra ones out there now, along with outdated ones being revised!
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -18,12 +18,34 @@ Maintenance is full of equipment that is randomized every round. Look around and
...
+Most things have special interactions with your middle mouse button, alt, shift, and control click. Experiment!
+You can screwdriver any non-chemical grenade to shorten fuses from 5 seconds, to 3 seconds, to 1 second, to instant boom! Nobody ever expects this.
+If you find yourself in a fistfight with another player, always stay on the offensive. Running away won't accomplish much.
+Different weapons have different strengths. Some weapons, such as spears, floor tiles, and throwing stars, deal more damage when thrown compared to when attacked normally.
+A thrown glass of water can make a slippery tile, allowing you to slow down your pursuers in a pinch.
+When dealing with security, you can often get your sentence negated entirely through cooperation and deception.
+The P2P chat function found on tablet computers allows for a stealthy way to communicate with people.
+Experiment with different setups of the supermatter engine to maximize output, but don't risk the crew's safety to do so!
+If you don't agree with an admin's ruling, make a policy discussion on the forums. It's what they're there for!
+Demoman takes skill.
```
Comment (Borisvanmemes) [inline strings/tips.txt]: But does he really?

File: strings/tips.txt [hunk 2]
```diff
@@ -18,12 +18,34 @@ Maintenance is full of equipment that is randomized every round. Look around and
...
+We were all new once, be patient and guide new players in the right direction.
+On most clothing items that go in the exosuit slot, you can put certain small items into your suit storage, such as a spraycan, your emergency oxygen tank, or a flashlight.
+Most job-related exosuit clothing can fit job-related items into it, such as the atmospheric technician's hardsuit/winter coat holding an RPD, or labcoats holding most medicine.
+If you're using hotkey mode, you can stop pulling things using H.
+The self-destruct is invincible. Go find the disk instead of trying to destroy it.
```
Comment (MrDoomBringer) [inline strings/tips.txt]:
```suggestion
The station's self-destruct terminal is invincible. Go find the disk instead of trying to destroy it.
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: strings/tips.txt (hunk 1 above)
Comment (plapatin) [reply strings/tips.txt]: ultimately it depends on if you use stickies as a primary substitute or for their intended purpose, aka traps and short-term area denial

Comment (optimumtact) [reply strings/tips.txt]: that should be in sillytips you muppet

File: strings/tips.txt [hunk 3]
```diff
@@ -18,12 +18,34 @@ Maintenance is full of equipment that is randomized every round. Look around and
...
+Most things have special interactions with your middle mouse button, alt, shift, and control click. Experiment!
+You can screwdriver any non-chemical grenade to shorten fuses from 5 seconds, to 3 seconds, to 1 second, to instant boom! Nobody ever expects this.
+If you find yourself in a fistfight with another player, always stay on the offensive. Running away won't accomplish much.
+Different weapons have different strengths. Some weapons, such as spears, floor tiles, and throwing stars, deal more damage when thrown compared to when attacked normally.
+A thrown glass of water can make a slippery tile, allowing you to slow down your pursuers in a pinch.
+When dealing with security, you can often get your sentence negated entirely through cooperation and deception.
+The P2P chat function found on tablet computers allows for a stealthy way to communicate with people.
+Experiment with different setups of the supermatter engine to maximize output, but don't risk the crew's safety to do so!
+If you don't agree with an admin's ruling, make a policy discussion on the forums. It's what they're there for!
```
Comment (optimumtact) [inline strings/tips.txt]: this should not be in the tips, please remove it

**Comment by yoyobatty** [comment]
Is there a way to see a random tip or list of tips? If not, there should be a way. Seeing a new tip every round is a very slow way of finding out useful ingame info.

File: strings/tips.txt [hunk 4]
```diff
@@ -18,12 +18,34 @@ Maintenance is full of equipment that is randomized every round. Look around and
...
+Don't be afraid to ask for help, whether from your peers or from admins.
 As the Captain, you are one of the highest priority targets on the station. Everything from revolutions, to nuclear operatives, to traitors that need to rob you of your unique lasgun or your life are things to worry about.
 As the Captain, always take the nuclear disk and pinpointer with you every shift. It's a good idea to give one of these to another head you can trust with keeping it safe.
 As the Captain, you have absolute access and control over the station, but this does not mean that being a horrible person won't result in mutiny and a ban.
-As the Captain, you can purchase a new emergency shuttle using a communications console. Some require cash, while others give you cash in exchange. Keep in mind that purchasing dangerous shuttles will incur the ire of your crew.
+As the Captain, you can purchase a new emergency shuttle using a communications console. Some require cash, while others give you cash in exchange. Keep in mind that purchasing dangerous shuttles, such as Oh Hi Daniel, will make you valid.
```
Comment (MrPerson) [inline strings/tips.txt]: Probably not a good idea to use "valid" in this context because a new player won't understand what it means. That's also somewhat of an administrative issue, which these tips should stay away from.

File: strings/tips.txt [hunk 5]
```diff
@@ -18,12 +18,34 @@ Maintenance is full of equipment that is randomized every round. Look around and
...
 As the Chief Medical Officer, your hypospray is like a refillable instant injection syringe that can hold 30 units as opposed to the standard 15.
 As the Chief Medical Officer, coordinate and communicate with your doctors, chemists, and geneticists during a nuclear emergency, blob infestation, or some other crisis to keep people alive and fighting.
+As the Chief Medical Officer, your hardsuit can hold an entire medkit in it's suit storage!
```
Comment (MrPerson) [inline strings/tips.txt]: its, not it's

File: strings/tips.txt [hunk 6]
```diff
@@ -71,11 +95,12 @@ As the Chief Engineer, your hardsuit is significantly better than everybody else
...
 As an Engineer, you can electrify grilles by placing wire "nodes" beneath them: the big seemingly unconnected bulges from a half completed wiring job.
 As an Engineer, return to Engineering once in a while to check on the engine and SMES cells. It's always a good idea to make sure containment isn't compromised.
-As an Engineer, you can power the station solely with the solar arrays. While uninteresting, it is a much safer alternative to regular engines.
+As an Engineer, you can temporarily power the station solely with the solar arrays. They will provide just enough electricity to power the station for a while, however their output is just slightly below what is truely needed to consistently power the station.
 As an Engineer, you can cool the supermatter crystal by spraying it with a fire extinguisher. Only for the brave!
+As an Engineer, if you become face-to-face with the threat of a blob, try setting up a PA launcher (like the kind used in singularities) in an important chokehold. The accelerated particles will severely weaken and eventually destroy that line, allowing your crewmembers to push the offensive. This tactic is ineffective against blob cores themselves, however.
```
Comment (MrPerson) [inline strings/tips.txt]: No need to abbreviate PA here, just say particle accelerator. Actually I would rephrase this to be shorter, something like

"As an Engineer, the particle accelerator used to power the gravitational singularity is also effective at killing blobs."

File: strings/tips.txt [hunk 7]
```diff
@@ -100,7 +126,7 @@ As the Warden, if a prisoner's crimes are heinous enough you can put them in per
 As the Warden, you can implant criminals you suspect might re-offend with devices that will track their location and allow you to remotely inject them with disabling chemicals. 
 As the Warden, you can use handcuffs on orange prisoner shoes to turn them into cuffed shoes, forcing prisoners to walk and potentially thwarting an escape.
 As a Security Officer, communicate and coordinate with your fellow officers using the security channel (:s) to avoid confusion.
-As a Security Officer, your sechuds or HUDsunglasses can not only see crewmates' job assignments and criminal status, but also if they are mindshield implanted. Use this to your advantage in a revolution to definitively tell who is on your side!
+As a Security Officer, your sechuds or HUDsunglasses can not only see crewmates' job assignments and criminal status, but also if they are mindshield implanted(You can tell if they have a flashing blue outline over their job icon). Use this to your advantage in a revolution to definitively tell who is on your side!
```
Comment (MrPerson) [inline strings/tips.txt]: Generally speaking, avoid using () if you can possibly get away with it.

File: strings/tips.txt [hunk 8]
```diff
@@ -112,11 +138,13 @@ As the Head of Personnel, you are not higher ranking than other heads of staff,
 As the Head of Personnel, you are just as large a target as the Captain because of the potential power your ID and computer can hand out.
 As the Mime, your invisible wall power blocks people as well as projectiles. You can use it in a pinch to delay your pursuer.
 As the Mime, you can use :r and :l to speak through your ventriloquist dummy.
-As the Mime, your oath of silence is your source of power. Breaking it robs you of your powers and of your honor.
+As the Mime, your oath of silence is your source of power. Breaking it robs you of your powers and of your honor, and might even make you valid, depending on the circumstance. A freshly-cloned Mime is not valid, as they lose their vow by default.
```
Comment (MrPerson) [inline strings/tips.txt]: "valid" stuff applies here, and frankly the freshly cloned stuff is so narrow that it makes for a shitty tip. That sounds more like behavior we should fix rather than warn people about tbh

File: strings/tips.txt [hunk 9]
```diff
@@ -125,9 +153,11 @@ As a Botanist, you can mutate the plants growing in your hydroponics trays with
 As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
 As a Botanist, you can combine production trait chemicals just like a Chemist. Chlorine (blumpkin) + radium and phosphorus (glowshrooms) equals unstable mutagen!
 As a Cook, you can load your food into snack vending machines.
+As a Cook, you can create a very wide variety of food with the crafting menu (that little T symbol next to your intents).
```
Comment (MrPerson) [inline strings/tips.txt]: It's supposed to be a hammer. Everyone thinks of it as a t though, because it really looks like one.

File: strings/tips.txt [hunk 10]
```diff
@@ -188,14 +218,6 @@ As a Cultist, your team starts off very weak, but if necessary can quickly conve
 As a Cultist, the Blood Boil rune will deal massive amounts of brute damage to non-cultists, stamina damage to Ratvarian scum, and some damage to fellow cultists of Nar'Sie nearby, but will create a fire where the rune stands on use.
 As a Cultist, you can create an army of manifested goons using a combination of the Manifest rune, which creates homunculi from ghosts, and the Blood Drain rune, which drains life from anyone standing on any blood drain rune.
 As a Cultist or Servant, check the alert in the upper-right of your screen for all the details about your cult's current status and objective.
```
Comment (MrPerson) [inline strings/tips.txt]: Remove the "or Servant" from this line too, as well as any other lines that might mention servant.

File: strings/tips.txt (hunk 1 above)
Comment (subject217) [reply strings/tips.txt]: grenade launcher spam that actually hits something is a highly skillful art

**Comment by ExcessiveUseOfCobblestone** [comment]
> Is there a way to see a random tip or list of tips? If not, there should be a way. Seeing a new tip every round is a very slow way of finding out useful ingame info.

looking at this file

**Comment by yoyobatty** [comment]
New players don't know or care about GitHub. They'd rather have a button to see a new tip, they're busy playing and learning the game.

**Comment by ExcessiveUseOfCobblestone** [comment]
They want to play the game but also want a popup window to read the list of 200+ tips...?

Have the wiki link the raw tips file in the neat tips section or something if players can't piece out how to navigate github (lol).

**Comment by yoyobatty** [comment]
That works too, the idea is to make it easily accessible to noobs because this provides extremely useful information.

**Comment by MrPerson** [comment]
Veteran players balk when people tell them to "read the code lol" and you expect a new player to do so? You insane bro.

Any suggestions of a rotating tip generator or give tip button are good ideas. Feel free to do them and make a pull request.

**Comment by Poojawa** [comment]
I still feel like a lot of these tips aren't nearly as vague as they could be. but then again I like thinking outside of boxes.

**Comment by yoyobatty** [comment]
MrPerson, I agee and even though I am a veteran much like cobblestone, I am the headmaintainer on hippie so the point is to enable as much user friendly options as possible so that the ss13 experience as a whole is just that much easier for new and returning players.

File: strings/tips.txt [hunk 11]
```diff
@@ -18,12 +18,32 @@ Maintenance is full of equipment that is randomized every round. Look around and
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
 You can use a machine in the vault to deposit cash or rob Cargo's department funds.
+When in doubt, clear your cache (byond launcher > cogwheel > preferences > game prefs), update your BYOND, and relog.
+Most things have special interactions with your middle mouse button, alt, shift, and control click. Experiment!
+You can screwdriver any non-chemical grenade to shorten fuses from 5 seconds, to 3 seconds, to 1 second, to instant boom! Nobody ever expects this.
+If you find yourself in a fistfight with another player, always stay on the offensive. Running away won't accomplish much.
```
Comment (Bawhoppen) [inline strings/tips.txt]: I would personally reword this one to something like:
"If you find yourself in a fistfight with another player, staying on the offensive can be the smart move. Running away often won't accomplish much."

File: strings/tips.txt [hunk 12]
```diff
@@ -18,12 +18,32 @@ Maintenance is full of equipment that is randomized every round. Look around and
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
 You can use a machine in the vault to deposit cash or rob Cargo's department funds.
+When in doubt, clear your cache (byond launcher > cogwheel > preferences > game prefs), update your BYOND, and relog.
```
Comment (Bawhoppen) [inline strings/tips.txt]: While it should be clear enough, maybe preface by noting "When in doubt about technical issues,"

File: strings/tips.txt [hunk 13]
```diff
@@ -18,12 +18,32 @@ Maintenance is full of equipment that is randomized every round. Look around and
...
+The P2P chat function found on tablet computers allows for a stealthy way to communicate with people.
+Experiment with different setups of the supermatter engine to maximize output, but don't risk the crew's safety to do so!
+We were all new once, be patient and guide new players in the right direction.
+On most clothing items that go in the exosuit slot, you can put certain small items into your suit storage, such as a spraycan, your emergency oxygen tank, or a flashlight.
+Most job-related exosuit clothing can fit job-related items into it, such as the atmospheric technician's hardsuit/winter coat holding an RPD, or labcoats holding most medicine.
+If you're using hotkey mode, you can stop pulling things using H.
+The station's self-destruct terminal is invincible. Go find the disk instead of trying to destroy it.
+If there's something you need from another department, try asking! This game isn't singleplayer and you'd be surprised what you can get accomplished together!
+You'll quickly lose your interest in the game if you play to win and kill. If you find yourself doing this, take a step back and talk to people - it's a much better experience!
+Ask and you shall receive.
```
Comment (Bawhoppen) [inline strings/tips.txt]: This is the normal tips list, and it seems kinda not very specific or clear what it's in reference to.

File: strings/tips.txt [hunk 14]
```diff
@@ -18,12 +18,32 @@ Maintenance is full of equipment that is randomized every round. Look around and
...
+Catgirls get temporarily distracted by laser pointers. Use this to your advantage when being pursued by one.
```
Comment (Bawhoppen) [inline strings/tips.txt]: Felinids/catpeople would probably be the preferred way to say this I think.

Comment (ExcessiveUseOfCobblestone) [reply strings/tips.txt]: lol

**Comment by ExcessiveUseOfCobblestone** [comment]
> Veteran players balk when people tell them to "read the code lol" and you expect a new player to do so? You insane bro.
> 
> Any suggestions of a rotating tip generator or give tip button are good ideas. Feel free to do them and make a pull request.

I don't really consider this file code anymore than the wiki. I can understand people being hesitant on reading actual CODE but not normal english.

I don't mind a rotating tip generator, but we already have that. If the tips are that imperative to read then it's wrong to handwave it by having it chance up at the beginning of the round, having a player click until they get the desired tip, or having them read the wiki/file.
